src/main/java/com/jvm_bloggers/core/rss/converters/SyndFeedXmlMessageConverter.java

```diff
@@ -0,0 +1,48 @@
+package com.jvm_bloggers.core.rss.converters;
+
+import com.rometools.rome.feed.synd.SyndFeed;
+import com.rometools.rome.io.SyndFeedOutput;
+
+import lombok.SneakyThrows;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpInputMessage;
+import org.springframework.http.HttpOutputMessage;
+import org.springframework.http.converter.AbstractHttpMessageConverter;
+import org.springframework.http.converter.HttpMessageNotReadableException;
+import org.springframework.http.converter.HttpMessageNotWritableException;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+
+import static org.springframework.http.MediaType.APPLICATION_ATOM_XML;
+
+@Component
+public class SyndFeedXmlMessageConverter extends AbstractHttpMessageConverter<SyndFeed> {
+
+    @Autowired
+    public SyndFeedXmlMessageConverter() {
+        super(APPLICATION_ATOM_XML);
+    }
+
+    @Override
+    protected boolean supports(Class<?> clazz) {
+        return SyndFeed.class.isAssignableFrom(clazz);
+    }
+
+    @Override
+    protected SyndFeed readInternal(Class<? extends SyndFeed> clazz, HttpInputMessage inputMessage)
+        throws IOException, HttpMessageNotReadableException {
+        throw new UnsupportedOperationException(
+            "Reading SyndFeed from a XML file is currently not supported!");
+    }
+
+    @Override
+    @SneakyThrows
+    protected void writeInternal(SyndFeed syndFeed, HttpOutputMessage outputMessage)
+        throws IOException, HttpMessageNotWritableException {
+
+        String xml = new SyndFeedOutput().outputString(syndFeed);
+        outputMessage.getBody().write(xml.getBytes());
```

Comment:
Sorry, I haven't noticed this before, but we can save some memory here. 
You can just create a `Writer` from `outputMessage` and pass it to `SyndFeedOutput`, something like:
```
Writer writer = new PrintWriter(outputMessage.getBody());
new SyndFeedOutpur().output());
```
This way we can skip conversion from Stream to String and back to Stream.